comparing_template/register/views.py

```diff
@@ -0,0 +1,15 @@
+from django.shortcuts import render, redirect, HttpResponse
+from .forms import RegisterForm
+from django.contrib.auth import views as auth_views
+
+def register(request):
+    if request.method == "POST":
+        form = RegisterForm(request.POST)
+        if form.is_valid():
+            form.save()
+            return redirect("login")  # Przekierowanie na stronę logowania
```

Comment:
ten komentarz jest zbędny, dodajemy to tylko wtedy jeśli kod sam w sobie nie jest zrozumiały, no i komentarze po angielsku jak cały kod